src/PAModel/Entropy.cs

```diff
@@ -56,6 +56,11 @@ internal class PropertyEntropy
 
         public PropertyEntropy VolatileProperties { get; set; }
 
+        // Sometimes, an empty LocalDatabaseReferences is serialized as "" or { }.
+        // Need to track which so we can round-trip.
+        // if true: "".   Else,  "{}". 
+        public bool LocalDatabaseReferencesAsEmpty { get; set; }
```

Comment:
Also, should this be nullable, for when the LocalDatabaseReference isn't empty at all?